scripts/makehtml.py

```diff
@@ -69,7 +69,8 @@ def parse_device(svdfile):
             rdesc = rtag.findtext('description')
             rrstv = rtag.findtext('resetValue')
             raccs = rtag.findtext('access') or "Unspecified"
-            roffset = int(rtag.findtext('addressOffset'), 16)
+            addr_offset = rtag.findtext('addressOffset')
+            roffset = int(addr_offset, 16 if addr_offset.startswith('0x') else 10)
```

Comment:
```suggestion
            roffset = int(rtag.findtext('addressOffset'), 0)
```

Unless we have any octal expressions I think this should work and is a bit simpler, what do you think?